### PR TITLE
Medical GUI - Add setting to show blood loss

### PR DESCRIPTION
**Pull request by BrettMayson**
**When merged this pull request will:**
- Title

### CODE REVIEW

File: addons/medical_gui/stringtable.xml [hunk 1]
```diff
@@ -1179,5 +1179,13 @@
             <German>Schadensfarbe %1</German>
             <Polish>Obrażenia - kolor %1</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_ShowBloodEntry_DisplayName">
+            <English>Show Blood Loss</English>
+            <German>Blutverlust anzeigen</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_GUI_ShowBloodlossEntry_Description">
+            <English>Show blood loss in the injury list.</English>
```
Comment (mharis001) [inline addons/medical_gui/stringtable.xml:1187]: I think including the "qualitative description" part from the comment in code may be an minor improvement.

File: addons/medical_gui/initSettings.sqf [hunk 2]
```diff
@@ -108,3 +108,11 @@ private _categoryColors = [ELSTRING(medical,Category), format ["| %1 |", LELSTRI
...
     ] call CBA_fnc_addSetting;
 } forEach _damageColors;
+
+[
+    QGVAR(showBloodlossEntry),
+    "CHECKBOX",
+    [LSTRING(ShowBloodlossEntry_DisplayName), LSTRING(ShowBloodlossEntry_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    true
+] call CBA_fnc_addSetting;
```
Comment (mharis001) [inline addons/medical_gui/initSettings.sqf:118]: Unsure on this but maybe we want to force this to be a global setting.

Comment (pterolatypus) [reply addons/medical_gui/initSettings.sqf:118]: Personally I think settings should only be global where absolutely necessary. There seems no reason not to just leave this up to server admins.

**Review by mharis001 — APPROVED** [review]

File: addons/medical_gui/stringtable.xml [hunk 3]
```diff
@@ -1179,5 +1179,13 @@
             <German>Schadensfarbe %1</German>
             <Polish>Obrażenia - kolor %1</Polish>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_ShowBloodEntry_DisplayName">
```
Comment (severgun) [inline addons/medical_gui/stringtable.xml:1182]:
```suggestion
        <Key ID="STR_ACE_Medical_GUI_ShowBloodlossEntry_DisplayName">
```

**Review by mharis001 — CHANGES_REQUESTED** [review]
Fix stringtable.

**Review by mharis001 — APPROVED** [review]

**Review by veteran29 — APPROVED** [review]

**Comment by BrettMayson** [comment]
Merge?

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**